presto-cassandra/src/main/java/com/facebook/presto/cassandra/NativeCassandraSession.java

```diff
@@ -171,6 +173,9 @@ public String getCaseSensitiveSchemaName(String caseInsensitiveSchemaName)
         for (TableMetadata table : keyspace.getTables()) {
             builder.add(table.getName());
         }
+        for (MaterializedViewMetadata materializedView : keyspace.getMaterializedViews()) {
```

Comment:
OK great, it's ok to not support the beta versions. So we don't need version checking here.